core/thread/inc/ROOT/TThreadedObject.hxx

```diff
@@ -286,7 +287,7 @@ namespace ROOT {
 
    };
 
-   template<class T> unsigned TThreadedObject<T>::fgMaxSlots = 64;
+   template<class T> unsigned TThreadedObject<T>::fgMaxSlots = std::max(std::thread::hardware_concurrency(), 64u);
```

Comment:
Why not simply use `std::thread::hardware_concurrency()`?

Reply:
For not breaking previous behaviour?

But yeah, I was told to keep a minimum of 32 🤷‍♂️